### ISSUE TITLE
version 2.1.0 in npm doesn't include index.js

### ISSUE THREAD

**Issue by yangaobo** [koajs/koa-redis#24]
I just installed the latest version `2.1.0` and failed to `require('koa-redis')`
And below is the output of the module
```
$ ls -al node_modules/koa-redis          
total 40
drwxr-xr-x    6 xxx  staff    204 May 19 15:21 .
drwxr-xr-x  797 xxx  staff  27098 May 19 15:21 ..
-rw-r--r--    1 xxx  staff   1320 May 19 11:30 History.md
-rw-r--r--    1 xxx  staff   1077 May 19 11:30 LICENSE
-rw-r--r--    1 xxx  staff   7726 May 19 12:00 README.md
-rw-r--r--    1 xxx  staff   2736 May 19 15:21 package.json
```

**Comment by coderhaoxin** [comment]
It seems a bug of npm in node@6.0.0 :).
Maybe need republish with node@5- or node@6.1+

**Comment by DaAwesomeP** [comment]
Oh I see what happened. I'll republish by the end of the day. Thanks!

**Comment by hujb2000** [comment]
Yes, koa-redis version 2.1.0 in npm doesn't include index.js, yet koa-redis version 2.0.1 include index.js that is ok

**Comment by DaAwesomeP** [comment]
I think that I ran into this bug: https://github.com/npm/npm/issues/5082.

@dead-horse how did you solve this issue last time?

**Comment by DaAwesomeP** [comment]
Published v2.1.1 by packing the packaged first before publishing.
